src/RichText.webmodeler.ts

```diff
@@ -6,50 +6,23 @@ import { RichTextContainerProps } from "./components/RichTextContainer";
 import { parseStyle } from "./utils/ContainerUtils";
 import { ValidateConfigs } from "./components/ValidateConfigs";
 
-interface PreviewState {
-    reloadEditor: boolean;
-}
-
 // tslint:disable-next-line class-name
-export class preview extends Component<RichTextContainerProps, PreviewState> {
-    constructor(props: RichTextContainerProps) {
-        super(props);
-
-        this.state = {
-            reloadEditor: false
-        };
-    }
-
+export class preview extends Component<RichTextContainerProps, {}> {
     render() {
-        return !this.state.reloadEditor
-            ? createElement(ValidateConfigs, { ...this.props as RichTextContainerProps, showOnError: true },
-                createElement(RichText, preview.transformProps(this.props)))
-            : null;
-    }
-
-    componentWillReceiveProps() {
-        this.setState({ reloadEditor: true });
-    }
-
-    componentDidUpdate() {
-        if (this.state.reloadEditor) {
-            this.setState({ reloadEditor: false });
-        }
+        return createElement(ValidateConfigs, { ...this.props as RichTextContainerProps, showOnError: true },
+            createElement(RichText, preview.transformProps(this.props))
+        );
     }
 
     private static transformProps(props: RichTextContainerProps): RichTextProps {
         const valueAttribute = props.stringAttribute ? props.stringAttribute.split(".")[ 2 ] : "";
 
         return {
+            ... props as any,
```

Comment:
Think any type should be solved differently.
Note that `hasContext` is not a prop of RichText

Reply:
Removed. Explicitly adding the props.